### PR TITLE
Navigation bar

### PR DESCRIPTION
**Pull request by kaitlynmcgoldrick**
- Replaced PageTitle with NavBar
- Added it to main page and report page

### CODE REVIEW

File: client/src/components/Image.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import StyledImage from './styled/StyledImage';
+// import StyledImage from './styled/StyledImage';
```
Comment (heathercodes) [inline client/src/components/Image.js:2]: Let's just delete this if we're not using it

File: client/src/components/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+import Image from './Image';
+import StyledNavBar from './styled/StyledNavBar';
+import { Navbar, Nav, NavItem } from 'react-bootstrap';
+
+const NavBar = ({ children }) => {
+    return (
+        <StyledNavBar>
+            <Navbar.Header>
+                <Navbar.Brand>
+                     <a href="/">
```
Comment (heathercodes) [inline client/src/components/NavBar.js:12]: Can we use the <Link> component here?

Comment (KristenLutz) [reply client/src/components/NavBar.js:12]: Is navbar both a custom component and a react-bootstrap component?

Comment (kaitlynmcgoldrick) [reply client/src/components/NavBar.js:12]: Ah yeah I see the naming convention is confusing.. Navbar is a react-bootstrap component and NavBar is a custom component. I can rename NavBar, any suggestions on what to rename it?

Comment (KristenLutz) [reply client/src/components/NavBar.js:12]: It's a bit confusing but if it works I'm not sure I have an issue with the naming, but if we are taking new names I vote CatNavBar

**Review by KristenLutz — APPROVED** [review]
Looks good!

**Pull request merged**